src/kayenta/metricStore/newrelic/metricConfigurer.tsx

```diff
@@ -16,7 +16,7 @@ interface INewRelicMetricConfigurerDispatchProps {
   changeSelect: (event: any) => void;
 }
 
-type INewRelicConfigurerProps = INewRelicMetricConfigurerStateProps & INewRelicMetricConfigurerDispatchProps;
+type NewRelicMetricConfigurerProps = INewRelicMetricConfigurerStateProps & INewRelicMetricConfigurerDispatchProps;
```

Comment:
Sorry for the nit but we try to keep all interfaces as starting with "I" per convention, so can we keep this as `INewRelicConfigurerProps` and change the reference to it below instead?

Reply:
Oh, I thought that was actually a mistake because it's a "type" not an interface?  No problem.

Reply:
Just looked into this a little bit and it seems like the current Microsoft-issued Typescript best practices say never to preface interfaces with the letter "I" at all: https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names
:cry: Since elsewhere in this codebase we preface interfaces, and types that are a union of interfaces, with I, I guess let's just continue to do that for consistency, but perhaps the UI SIG should revisit this... :man_shrugging:

Reply:
I've only seen examples in deck/deck-kayenta where we preface with `I` on unions of two interfaces *and no other type expressions* — which to me feels reasonable but certainly isn't authoritative. @maggieneterval have you seen other places where types get an `I` prefix and are *not* unions of interfaces?

Reply:
> @maggieneterval have you seen other places where types get an I prefix and are not unions of interfaces?

Nope, and agreed that it feels reasonable to preface with `I` on unions of two interfaces _and no other type expressions_ :+1: